msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/SignInStates.kt

```diff
@@ -53,6 +53,7 @@ import com.microsoft.identity.common.java.util.StringUtil
 import com.microsoft.identity.common.java.nativeauth.util.checkAndWrapCommandResultType
 import com.microsoft.identity.nativeauth.statemachine.errors.ErrorTypes
 import com.microsoft.identity.nativeauth.statemachine.errors.ResendCodeError
+import com.microsoft.identity.nativeauth.statemachine.errors.SignInContinuationError
```

Comment:
SignInPasswordRequiredState loses Parcelable implementation.

Reply:
@SaurabhMSFT, what is the reason this class does not implement Parcelable?

Reply:
I did this in my PR: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2007/files